### PR TITLE
Fix Navbar Text Position and Height Inequality in Mobile Bottom Navbar

### LINKED ISSUE #510: Fix Navbar Text Position and Height Inequality in Mobile Bottom Navbar
### Description

# Issue Title: Navbar Text Position and Height Inequality in Mobile Bottom Navbar

## Issue Description:
This consolidated issue addresses two related concerns with the website's navigation bar (navbar):

### **Issue 1**: Navbar Text Position Issue on Larger Devices

- On larger devices, when hovering over the navbar menus, the text doesn't appear in the same position as the navbar menu. Instead, it's displayed slightly higher, which affects the visual alignment.

### **Issue 2**: Unequal Height of Bottom Navbar on Mobile Devices

- On mobile devices, when the navbar is transformed into a bottom navbar, the height of the navbar buttons becomes unequal. For example, when clicking on "Search," the button's height is different from when clicking on "Profile."

## Issue 1
image

## Issue 2
image

image

### Additional information

As I'm creating this issue in the repository I want to contribute to, I'm open to receiving specific guidelines or preferences regarding the resolution of these issues. Your input will help ensure that the proposed changes align with the project's goals and standards.

### 🥦 Browser

Microsoft Edge

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by purnasth**
# Pull Request: Fix Navbar Text Position and Height Inequality

Fixes #510 

### Issue 1: Navbar Text Position Issue
image

### Issue 2: Unequal Height of Bottom Navbar on Mobile Devices
image

image

### CODE REVIEW

**Comment by Dun-sin** [comment]
> # PR Compliance Checks
> 
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> ## Conventional Commit PR Title
> 
> In order to be considered for merging, the pull request title must match the specification in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass. Most often, our PR titles are something like one of these:
> 
>     * docs: correct typo in README
> 
>     * feat: implement dark mode"
> 
>     * fix: correct remove button behavior
> 
> 
> Linting Errors
> 
>     * Found type "null", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"
> 
>     * No subject found

@purnasth Please fix this error, if you don't know how, try this:
- Read the error carefully or read the contributing file
- Checkout past merged Pull requests to see how they were done for it to be merged

> Note: you can edit your PR message, this PR won't be merged if the error isn't gone and you have a week max to fix it before the PR gets closed without being merged

**Comment by Dun-sin** [comment]
@purnasth we have templates for a reason, use them properly or don't create a PR/Issue at all, it's annoying

File: client/src/components/NavBar.jsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ const NavBar = ({ className }) => {
 					placement="auto"
 					controlId="control-id-hover"
 					trigger="hover"
-					speaker={<Tooltip>Logout</Tooltip>}
+					speaker={<Tooltip className="md:-mt-4">Logout</Tooltip>}
```
Comment (Dun-sin) [inline client/src/components/NavBar.jsx:136]: not part of your issue, remove it

File: client/src/components/NavBar.jsx [hunk 2]
```diff
@@ -123,7 +123,7 @@ const NavBar = ({ className }) => {
 					placement="auto"
 					controlId="control-id-hover"
 					trigger="hover"
-					speaker={<Tooltip>Settings</Tooltip>}
+					speaker={<Tooltip className="md:-mt-4">Settings</Tooltip>}
```
Comment (Dun-sin) [inline client/src/components/NavBar.jsx:126]: not part of your issue, remove it

File: client/src/components/NavBar.jsx [hunk 3]
```diff
@@ -113,7 +113,7 @@ const NavBar = ({ className }) => {
 					placement="auto"
 					controlId="control-id-hover"
 					trigger="hover"
-					speaker={<Tooltip>My Profile</Tooltip>}
+					speaker={<Tooltip className="md:mt-4">My Profile</Tooltip>}
```
Comment (Dun-sin) [inline client/src/components/NavBar.jsx:116]: not part of your issue, remove it

File: client/src/components/NavBar.jsx [hunk 4]
```diff
@@ -103,7 +103,7 @@ const NavBar = ({ className }) => {
 					placement="auto"
 					controlId="control-id-hover"
 					trigger="hover"
-					speaker={<Tooltip>Friends</Tooltip>}
+					speaker={<Tooltip className="md:mt-4">Friends</Tooltip>}
```
Comment (Dun-sin) [inline client/src/components/NavBar.jsx:106]: not part of your issue, remove it

File: client/src/components/NavBar.jsx [hunk 5]
```diff
@@ -84,15 +84,15 @@ const NavBar = ({ className }) => {
...
+				'min-h-[70px]',
 			])}
 		>
 			<div className="justify-between md:justify-center flex items-center md:flex-col flex-row h-full w-full gap-2 flex-nowrap overflow-auto md:h-full">
 				<Whisper
 					placement="auto"
 					controlId="control-id-hover"
 					trigger="hover"
-					speaker={<Tooltip>Search for random buddies</Tooltip>}
+					speaker={<Tooltip className="md:mt-4">Search for random buddies</Tooltip>}
```
Comment (Dun-sin) [inline client/src/components/NavBar.jsx:95]: not part of your issue, remove it

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by purnasth** [comment]
I genuinely wanted to contribute to your project and enhance your website. The intention was to learn and help improve the project collaboratively. Your feedback is valued, but there are more constructive ways to convey it, you didn't have to be rude. We're all here to grow and learn, therefore it's critical to support and mentor newcomers. I don't think contributions from beginners are appreciated here. I appreciate your time and wish you continued success with your project.

**Comment by Dun-sin** [comment]
> I genuinely wanted to contribute to your project and enhance your website. The intention was to learn and help improve the project collaboratively. Your feedback is valued, but there are more constructive ways to convey it, you didn't have to be rude. We're all here to grow and learn, therefore it's critical to support and mentor newcomers. I don't think contributions from beginners are appreciated here. I appreciate your time and wish you continued success with your project.

appologies if anything i said was rude was just expressing my fraustration, considering i told you to read the rules and you didn't. 

that being said, are you closing the PR?

**Pull request closed without merging**